### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by max-bertinetti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,10 +7,19 @@
 After you install Homebrew, run the following command:
 
 ```
-brew tap homebrew-community/mint
+brew tap mint-lang/homebrew-mint-lang
```
Comment (SeekingMeaning) [inline README.md:10]:
```suggestion
brew tap mint-lang/mint-lang
```

**Review by SeekingMeaning — APPROVED** [review]
ping @gdotdesign

**Comment by gdotdesign** [comment]
Thanks :+1:

File: README.md (hunk 1 above)
Comment (gdotdesign) [reply README.md:10]: I would prefer this as well but, the repo name needs to start with `homebrew` unfortunately.

Comment (SeekingMeaning) [reply README.md:10]: `brew tap mint-lang/mint-lang` clones `mint-lang/homebrew-mint-lang`

Comment (gdotdesign) [reply README.md:10]: Nice, didn't know about that :+1:

**Pull request merged**